zebra/zebra_router.c

```diff
@@ -31,6 +31,8 @@
 #include "zebra_nhg.h"
 #include "debug.h"
 
+
+
```

Comment:
Remove this

Reply:
updated in https://github.com/FRRouting/frr/pull/9440/commits/a2df35f1459b554daff2c41507d16c66d1c0de50